src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,343 @@
+
+
+| Field name  | Definition                                                                                                      |
+| :---------- | :-------------------------------------------------------------------------------------------------------------- |
+| ColumnName    | REQUIRED. Name of the column (e.g. "time".                                                                        |
+| Description | REQUIRED. Time in relation to TimeZero defined by the *_pet.json. (e.g. 5)                                                                                      |
+| Units       | REQUIRED. Unit of time steps (e.g. "s") |   
+
+### Radioactivity in plasma
+This section may be omitted if plasma measurements of radioactivity were not made. It contains information regarding sampled plasma data. If plasma measurements are available by a discrete sampler e.g. using a COBRA counter, they should be added to the *recording-blood_discrete.tsv and the values below should be added to the *recording-blood_discrete.json. If plasma measurements are available by a continuous sampler e.g. using an Allogg autosampler, they should be added to the *recording-blood_continous.tsv and the values below should be added to the *recording-blood_continous.json.
+
+| Field name  | Definition                                                                                                      |
+| :---------- | :-------------------------------------------------------------------------------------------------------------- |
+| ColumnName    | REQUIRED. Name of the column (e.g. "plasma_radioactivity".                                                                        |
+| Description | REQUIRED. Radioactivity in plasma.                                                                                      |
+| Units       | REQUIRED. Unit of plasma radioactivity (e.g. "kBq/mL") |   
+
+### Metabolites
+This section may be omitted if metabolite measurements were not made. If metabolite measurements are available they should be added to the *recording-blood_discrete.tsv and the values below should be added to the *recording-blood_discrete.json.
+Hence, it may contain information regarding metabolite info, such as the following three column examples:
+
+
+| Field name  | Definition                                                                                                      |
+| :---------- | :-------------------------------------------------------------------------------------------------------------- |
+| ColumnName    | REQUIRED. Name of the column (e.g. "metabolite_parent_fraction".                                                                        |
+| Description | REQUIRED. Parent fraction of the radiotracer (0-1).                                                                                      |
+| Units       | REQUIRED. Unit of parent fraction (e.g. "unitless") |  
```

Comment:
Please replace "unitless" with "arbitrary" (see #606).

Reply:
Thanks @tsalo, we are changing this.